sgm/modules/diffusionmodules/lpips/lpips.py

```diff
@@ -0,0 +1,147 @@
+"""Stripped version of https://github.com/richzhang/PerceptualSimilarity/tree/master/models"""
```

Comment:
https://github.com/richzhang/PerceptualSimilarity/blob/master/LICENSE

Reply:
Good point should add it